### PR TITLE
P1 grid

### PR DESCRIPTION
**Pull request by Muzammil-khan710**
I have added grid component

### CODE REVIEW

File: docs/components/Grid/grid.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/styles.css">
+    <title>Navigation</title>
+</head>
```
Comment (Shubhamkr2610) [inline docs/components/Grid/grid.html]: change the title name

**Review by Shubhamkr2610 — APPROVED** [review]
rest everything looks good to me . you can merge it

File: docs/components/Grid/grid.html (hunk 1 above)
Comment (Muzammil-khan710) [reply docs/components/Grid/grid.html]: Changing

File: docs/components/Grid/grid.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
+.grid-two-items {
+    display: grid;
+    gap: 1rem;
+    grid-template-columns: auto auto;
+}
+  
+.grid-three-items {
+    display: grid;
+    gap: 1rem;
+    grid-template-columns: 1fr 1fr 1fr;
```
Comment (amartya-rs) [inline docs/components/Grid/grid.css]: suggestion: can use repeat function, grid-template-columns: repeat(3, 1fr)

File: docs/components/Grid/grid.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
+.grid-two-items {
+    display: grid;
+    gap: 1rem;
+    grid-template-columns: auto auto;
```
Comment (amartya-rs) [inline docs/components/Grid/grid.css]: suggestion: grid-template-columns: repeat(2, 1fr) would also work

**Review by sudipkundu999 — APPROVED** [review]
The code and UI look good.
Good to merge

**Review by amartya-rs — APPROVED** [review]
Code looks fine. You can merge.

**Pull request merged**